src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
+@import './variebles';
+@import './mixins';
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    @include font-fam('Roboto, sans-serif;', 12px, 500, 18px);
+
+    margin: 0 0 4px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    @include font-fam('Roboto, sans-serif;', 10px, 400, 14px);
+
+    margin: 0 0 16px;
+    color: $secondary-text-color;
+  }
+
+  &__button {
+    @include font-fam('Roboto, sans-serif;', 14px, 700, 16px);
```

Comment:
The font family 'Roboto, sans-serif;' is repeated many times in your CSS. Consider creating a variable for it.